shell/browser/ui/message_box_mac.mm

```diff
@@ -17,6 +17,27 @@
 #include "skia/ext/skia_utils_mac.h"
 #include "ui/gfx/image/image_skia.h"
 
+namespace {
+
+// Create an accessory view for richer dialogs.
+NSTextView* CreateAccessoryView(const std::string& content,
+                                int width,
+                                int height) {
+  NSTextView* accessory =
+      [[NSTextView alloc] initWithFrame:NSMakeRect(0, 0, width, height)];
```

Comment:
`NSView.accessoryView` is a `@property(strong)`, so this needs to be autoreleased.